pages/api/users/popular.js

```diff
@@ -7,7 +7,7 @@ import Profile from "../../../models/Profile";
 export default async function handler(req, res) {
   await connectMongo();
 
-  const getProfiles = await Profile.find({}).sort({ views: -1 }).limit(10);
+  const getProfiles = await Profile.find({}).sort({ views: -1 }).limit(50);
```

Comment:
I can't get the purpose of these changes.

Reply:
@ManishBisht777 these changes has already been merged, I just created another pr for the images since the text were making it look bad, like I pushed to another branch can you please check it ?

Reply:
ig I should have first closed this pr and than create another pr right ?
should I do that now ?

Reply:
You can remove it from your Git repository, which should show here.

Reply:
Delete it from here:
![image](https://user-images.githubusercontent.com/19922556/210737653-553e37a6-0b77-4c8c-b04c-17ea7efff85c.png)